share/mk/src.opts.mk

```diff
@@ -379,8 +379,8 @@ BROKEN_OPTIONS+=SVN SVNLITE
 BROKEN_OPTIONS+=LIBCHERI
 .endif
 
-# EFI doesn't exist on mips, powerpc or riscv.
-.if ${__T:Mmips*} || ${__T:Mpowerpc*} || ${__T:Mriscv*}
+# EFI doesn't exist on mips or powerpc.
```

Comment:
This breaks the build for purecap worlds. What should we do? Considering that the EFI code won't save capregs marking it as BROKEN_OPTIONS is probably the easiest solution?
I'd be tempted to drop https://reviews.freebsd.org/D25135 from this commit for now and revisit the problem when we merge it?

Reply:
In fact it seems broken for all RISC-V targets. I'm just going to drop that commit for now.